### PR TITLE
fix: dom ready script didn't return the value of the executed script

### PR DESCRIPTION
**Pull request by olivier-martin-sf**
This PR fixes an issue where the DOM_READY_JAVASCRIPT was executed by didn't explicitly return a value. As such, `null` was being returned while executing the script, which translated into a falsy value and the waitFor kept looping until it times out.

### CODE REVIEW

File: pom.xml [hunk 1]
```diff
@@ -12,7 +12,7 @@
     <groupId>com.salesforce.utam</groupId>
     <artifactId>utam-java</artifactId>
     <packaging>pom</packaging>
-    <version>1.1.5-SNAPSHOT</version>
```
Comment (jimevanssfdc) [inline pom.xml:15]: Revert the changes to the version property here. It needs to be `-SNAPSHOT` until we decide to release.

File: utam-core/src/main/java/utam/core/framework/element/DocumentObject.java [hunk 2]
```diff
@@ -28,7 +28,7 @@
  */
 public class DocumentObject implements Document {
 
-  static final String DOM_READY_JAVASCRIPT = "document.readyState === 'complete'";
+  static final String DOM_READY_JAVASCRIPT = "return document.readyState === 'complete'";
```
Comment (jimevanssfdc) [inline utam-core/src/main/java/utam/core/framework/element/DocumentObject.java:31]: Adding the `return` is correct. It's a shame we didn't catch this before.

**Review by jimevanssfdc — CHANGES_REQUESTED** [review]

File: pom.xml (hunk 1 above)
Comment (olivier-martin-sf) [reply pom.xml]: fixed in 02b97836759131ae068d7e2fa86597ab624cdae6

**Review by jimevanssfdc — APPROVED** [review]
